www/src/components/Header.tsx

```diff
@@ -0,0 +1,78 @@
+// Libraries
+import React, { FC } from 'react';
+
+// Material UI
+import makeStyles from '@material-ui/core/styles/makeStyles';
+import Toolbar from '@material-ui/core/Toolbar';
+import Link from '@material-ui/core/Link';
+import Container from '@material-ui/core/Container';
+
+// Assets
+import logo from '../assets/logo_white.svg';
+
+// Components
+import OutlinedButton from './OutlinedButton';
+
+type HeaderProps = {
+    sections: Section[];
+    title: string;
+};
+
+export type Section = {
+    title: string;
+    url: string;
+};
+
+export const Header: FC<HeaderProps> = (props: HeaderProps) => {
+    const classes = useStyles();
+    const { sections, title } = props;
+
+    return (
+        <Toolbar className={classes.toolbar} disableGutters={true}>
+            <img src={logo} className={classes.logo} />
+            <Link underline='none' color='inherit' noWrap variant='h5' href={'/'} className={classes.title}>
+                {title}
+            </Link>
+            <Container className={classes.linkContainer}>
+                {sections.map((section) => (
+                    <Link color='inherit' noWrap key={section.title} variant='body2' href={section.url} className={classes.toolbarLink}>
+                        {section.title}
+                    </Link>
+                ))}
+            </Container>
+            <OutlinedButton title={'Login'} url={'#'} />
+        </Toolbar>
+    );
+};
+
+const useStyles = makeStyles((theme) => ({
+    toolbar: {
+        boxShadow: '0px 5px 5px #547d54',
+        paddingLeft: 10,
+        paddingRight: 10,
+        backgroundColor: theme.palette.primary.main,
+        display: 'flex',
+        justifyContent: 'center',
+    },
```

Comment:
Any reason we're styling this manually instead of using their `AppBar` component with `position='relative'`?

Reply:
I followed an example from the Material UI website, didn't know Material UI has an `AppBar` component haha, will change it to use it instead.

Reply:
Alright alright 😂